src/components/ProductHero.tsx

```diff
@@ -3,21 +3,26 @@ import { FunctionComponent, ReactNode, useEffect, useRef } from 'react'
 import { Background, HubSpotForm } from '@components'
 
 interface ProductHero extends Background {
-    product: 'code search' | 'batch changes' | 'code insights'
+    product?: 'code search' | 'batch changes' | 'code insights'
     title: string | ReactNode
-    description: string
+    backButton?: ReactNode
+    description?: string
     displayUnderNav?: boolean
 }
```

Comment:
Since the `ProductHero` renders a HubSpot form component below the description, we may need to modify the API here slightly to accomodate the last outstanding props we have left to implement for the other Hero variations. How do you feel about a cta prop that allows you to include any `ReactNode` which will render under the description? That way we can include a HubSpot form or a set of CTA buttons.


```suggestion
    cta?: ReactNode
}
```

Or we could be explicit with some type of union prop so that we don't have to manually add CTA buttons every time for other hero's. I'm unsure if hero's on product pages will ever change the type of HubSpot form we use.